.github/workflows/gradle.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Build and check with Gradle
         run: ./gradlew check coverage
 
-      - uses: codecov/codecov-action@v1
+      - uses: codecov/codecov-action@v2
         if: runner.os == 'Linux'
         with:
           file: ${{ github.workspace }}/build/reports/jacoco/coverage/coverage.xml
```

Comment:
Did a quick scan through the [codecov README](https://github.com/codecov/codecov-action) and realized that the `file` argument might have been deprecated.

This argument was removed from the README over a year ago in this [commit](https://github.com/codecov/codecov-action/commit/25b873867a4d9c029691df2b437b0c7c7ba410cd).

Didn't have time to check if `file` is still working correctly. Should we consider shifting over to `files` instead?

Reply:
I tried out the `files` arg. It seems to work.